src/Refactoring-Core/RBExtractMethodAndOcurrences.class.st

```diff
@@ -0,0 +1,96 @@
+Class {
+	#name : #RBExtractMethodAndOcurrences,
```

Comment:
Could rename the class because there is a typo in the name Occurrence.
Could we get a class comment to indicate what the refactoring is actually doing?